lib/vnet/daemon/client_darwin.m

```diff
@@ -0,0 +1,99 @@
+// Teleport
```

Comment:
Repeat the Go build tags here so you don't get other platforms trying to compile this code.

```suggestion
//go:build darwin
// +build darwin

// Teleport
```

Reply:
Why are those repeated build tags needed in `_darwin.m` files but not in `_darwin.go` files? From what I tested, `_darwin.m` was enough to avoid compiling this on other platforms. Without the `_darwin` part, cgo on Linux wouldn't let me compile `lib/vnet/daemon`, even if there was no Go file on Linux that included this Obj-C file.

Reply:
I wasn't sure about .m, but if it works feel free to resolve the comment.